src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -0,0 +1,166 @@
+           // justify-content: center;
+            align-items: center;
+            overflow: hidden;
+            padding: $interiorMargin;
+
+            > * {
+                // Text elements
+                @include ellipsize();
+            }
+        }
+
+        > * + * {
+            margin-left: $interiorMargin;
+        }
+
+        .c-frame & {
+            @include abs();
+            border: 1px solid transparent;
+        }
+    }
+</style>
+
+ <script>
+    export default {
+        inject: ['openmct'],
+        props: {
+            item: Object
+        },
+        computed: {
+            showLabel: function () {
+                let displayMode = this.item.config.alphanumeric.displayMode;
+                return (displayMode === 'all' || displayMode === 'label') ? true : false;
+            },
+            showValue: function () {
+                let displayMode = this.item.config.alphanumeric.displayMode;
+                return displayMode === 'value' || displayMode == 'all' ? true : false;
+            },
+            styleObject: function () {
+                let alphanumeric = this.item.config.alphanumeric;
+                return {
+                    backgroundColor: alphanumeric.fill,
+                    borderColor: alphanumeric.stroke,
+                    color: alphanumeric.color,
+                    fontSize: alphanumeric.size
+                }
+            }
+        },
+        data() {
+            return {
+                telemetryValue: '',
+                telemetryClass: ''
+            }
+        },
+        methods: {
+            getTelemetry(domainObject) {
+                return Promise.resolve(domainObject)
+                    .then(this.fetchHistoricalData)
+                    .then(this.subscribeToObject);
+            },
+            fetchHistoricalData(object) {
+                let bounds = this.openmct.time.bounds();
+                this.openmct.telemetry.request(object, {start: bounds.start, end: bounds.end, size: 1})
+                    .then(function (data) {
+                        if (data.length > 0) {
+                            this.updateView(object, data[data.length - 1]);
+                        }
+                    }.bind(this));
+
+                return object;
+            },
+            subscribeToObject(object) {
+                let id = this.openmct.objects.makeKeyString(object.identifier);
+                let self = this;
+                this.subscription = this.openmct.telemetry.subscribe(object, function (datum) {
+                    if (self.openmct.time.clock() !== undefined) {
+                        self.updateView(object, datum);
+                    }
+                }, {});
+
+                return object;
+            },
+            updateView(telemetryObject, datum) {
+                let metadata = this.openmct.telemetry.getMetadata(telemetryObject);
+                let valueMetadata = metadata.values().filter(function (value) {
```

Comment:
1. can get metadata for value key with `metadata.value(key)`
2. should cache formatter and limit evaluator instead of re-evaluating this logic for every new data point that comes in.